warehouse/templates/includes/input-recaptcha.html

```diff
@@ -0,0 +1,14 @@
+{% if request.find_service(name="recaptcha").enabled %}
+  {% block top_script %}
+  <script src="www.google.com/recaptcha/api.js"></script>
```

Comment:
I guess recaptcha doesn't work if it's not included at the top? Otherwise it'd be preferable to include it at the end of the body to prevent blocking the rendering.